### PR TITLE
Add Coloban

### PR DESCRIPTION
**Pull request by pikil**
**By submitting this pull request, I promise I have read the [contributing guidelines](https://github.com/sindresorhus/awesome-electron/blob/master/contributing.md) twice and ensured my submission follows it. I realize not doing so wastes the maintainers' time that they could have spent making the world better. 🖖**

The app is a 'closed source' and built with using VueJS (https://www.coloban.com) and Quasar (https://github.com/quasarframework/quasar-awesome#projects-using-quasar).

The Desktop (Electron based) version in available at https://www.coloban.com/download (MacOS and Windows)

Many thanks!

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -247,6 +247,7 @@ Made with Electron.
 - [Notion](https://notion.so) - All-in-one workspace for your notes, tasks, wikis, and databases.
 - [FATpick](https://www.fatpick.com) - Guitar tablature viewer that scores your performance as you play along.
 - [Taskade](https://www.taskade.com) - Realtime organization and collaboration tool for distributed teams with tasks, notes, and chat.
+- [Coloban](https://www.coloban.com) - Save hours per week on Project Management with Chats, Screenshare, Kanban, Gantt and many more.
```
Comment (sindresorhus) [inline readme.md:250]: The description should be an objective description of the app, not a marketing tagline.

Comment (pikil) [reply readme.md]: Hi Sindre,
Sure, sorry - have corrected the description.

File: readme.md [hunk 2]
```diff
@@ -247,6 +247,7 @@ Made with Electron.
 - [Notion](https://notion.so) - All-in-one workspace for your notes, tasks, wikis, and databases.
 - [FATpick](https://www.fatpick.com) - Guitar tablature viewer that scores your performance as you play along.
 - [Taskade](https://www.taskade.com) - Realtime organization and collaboration tool for distributed teams with tasks, notes, and chat.
+- [Coloban](https://www.coloban.com) - All-in-one Project management tool with Chats, Kanban, Gantt, Calls, Screenshare and many more.
```
Comment (sindresorhus) [inline readme.md:250]: The description should not be title-case.

Comment (pikil) [reply readme.md]: Sure, have changed the description to match that :)

**Pull request merged**